### PR TITLE
Fixing URL to Grok patterns

### PR DESCRIPTION
**Pull request by jtyr**
This PR is just fixing the link to the Grok reusable patterns.

### CODE REVIEW

File: docs/reference/ingest/ingest-node.asciidoc [hunk 1]
```diff
@@ -1217,7 +1217,7 @@ expression that supports aliased expressions that can be reused.
 This tool is perfect for syslog logs, apache and other webserver logs, mysql logs, and in general, any log format
 that is generally written for humans and not computer consumption.
 This processor comes packaged with over
-https://github.com/elastic/elasticsearch/tree/master/modules/ingest-common/src/main/resources/patterns[120 reusable patterns].
+https://github.com/elastic/elasticsearch/blob/master/libs/grok/src/main/resources/patterns/grok-patterns[120 reusable patterns].
```
Comment (cbuescher) [inline docs/reference/ingest/ingest-node.asciidoc]: Nit: a slight improvement would be to link to the grok-patterns file on the current branch (either master or one of the version branches, e.g. 6.x, 6.2). You can use the `{branch}` placeholder for this inside the link as I just learned [recently](https://github.com/elastic/elasticsearch/pull/28530#issuecomment-363725730)

**Review by cbuescher — CHANGES_REQUESTED** [review]
@jtyr thanks a lot for fixing the broken link, I left a small suggestion for improvement.

File: docs/reference/ingest/ingest-node.asciidoc (hunk 1 above)
Comment (jtyr) [reply docs/reference/ingest/ingest-node.asciidoc]: Thanks for your suggestion. It's fixed now.

File: docs/reference/ingest/ingest-node.asciidoc [hunk 2]
```diff
@@ -1217,7 +1217,7 @@ expression that supports aliased expressions that can be reused.
 This tool is perfect for syslog logs, apache and other webserver logs, mysql logs, and in general, any log format
 that is generally written for humans and not computer consumption.
 This processor comes packaged with over
-https://github.com/elastic/elasticsearch/tree/master/modules/ingest-common/src/main/resources/patterns[120 reusable patterns].
+https://github.com/elastic/elasticsearch/blob/{branch}/libs/grok/src/main/resources/patterns/grok-patterns[120 reusable patterns].
```
Comment (cbuescher) [inline docs/reference/ingest/ingest-node.asciidoc]: Sorry about nitpicking, but when I re-checked the target directory for the link, I saw that maybe the parent directory (https://github.com/elastic/elasticsearch/tree/master/libs/grok/src/main/resources/patterns) would be a better choice? I think the link originally pointed there (e.g. on 5.6 branch: https://github.com/elastic/elasticsearch/tree/5.6/modules/ingest-common/src/main/resources/patterns)
Maybe @mvg would also know if the number of patterns should be adapted while we are at it. Looks like way more thank 120 to me, are they all packaged with the grok processor or only a subset of them?

**Comment by cbuescher** [comment]
@jtyr thanks for the update, I was about to merge but found another tiny thing to at least check while changing this, left a question for @mvg for clarification

**Review by cbuescher — APPROVED** [review]
Thanks, LGTM
Will run final tests before merging.

**Comment by cbuescher** [comment]
@elasticmachine test this please

**Pull request merged**